config/locales/es/settings.yml

```diff
@@ -130,6 +130,8 @@ es:
       dashboard:
         notification_emails: "Emails del panel de progreso"
         notification_emails_description: "Activar el envío de emails de notificaciones a los autores de las propuestas en la sección de panel de progreso"
+      sdg: ODS
+      sdg_description: Habilitar secciones relacionadas con Objetivos y Metas de Desarrollo Sostenible en el menú de administración y en la sección de Configuración Global.
```

Comment:
We mention "Metas" here but don't mention "Targets" en English :thinking:. Removing "Metas" is fine with me.